src/content/pages/faq.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: FAQ
+subtitle: Frequently Asked Questions about EuroPython
+---
+
+# FAQ
+
+## **Q. When and where is EuroPython 2025 taking place?**
+
+A. EuroPython 2025 will take place on **14–20 July 2025**. Mark your calendars!
```

Comment:
@cybit would you like to replace this with en-dash too?

Reply:
I think I applied it now in https://github.com/EuroPython/website/pull/949/commits/f66b84355d92b319c68fb692fbfc1742cd46e515 <3